code/modules/mob/living/carbon/human/human.dm

```diff
@@ -105,42 +105,66 @@
 		if (HUDtech.Find("flash"))
 			FLICK("flash", HUDtech["flash"])
 
-	var/shielded = 0
```

Comment:
What could go wrong?

Reply:
as far as I can tell nothing, especially as this isn't used, but if something does we should find out in the test server run.